consensus/src/block_storage/block_tree.rs

```diff
@@ -277,6 +397,12 @@ impl BlockTree {
         if new_commit_cert.commit_info().round() > self.highest_commit_cert.commit_info().round() {
             self.highest_commit_cert = Arc::new(new_commit_cert);
             self.update_commit_root(self.highest_commit_cert.commit_info().id());
+        } else {
+            warn!(
```

Comment:
this seems fine, is this an invariant we try to maintain?

Reply:
Can remove